cupy/_indexing/generate.py

```diff
@@ -472,10 +472,63 @@ def mask_indices(n, mask_func, k=0):
 # TODO(okuta): Implement mask_indices
 
 
-# TODO(okuta): Implement tril_indices
+def tril_indices(n, k=0, m=None):
+    """Returns the indices of the lower triangular matrix.
+    Here, the first row contains row coordinates of all indices and
+    the second row contains column coordinates.
+
+    Parameters
+    ----------
+    n : int
+        The row dimension of the arrays for which the returned
+        indices will be valid.
+    k : int, optional
+        Diagonal above which to zero elements. `k = 0`
+        (the default) is the main diagonal, `k < 0` is
+        below it and `k > 0` is above.
+    m : int, optional
+        The column dimension of the arrays for which the
+        returned arrays will be valid. By default, `m = 0`.
```

Comment:
Be default, `m` is taken to equal to `n`?

Reply:
Oops, typo. Thanks!